presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -433,7 +434,7 @@ private static LongWriteFunction timestampWithTimeZoneWriteFunction()
         return (statement, index, value) -> {
             // PostgreSQL does not store zone information in "timestamp with time zone" data type
             long millisUtc = unpackMillisUtc(value);
-            statement.setTimestamp(index, new java.sql.Timestamp(millisUtc));
```

Comment:
word of explanation: i sometimes tend to use `java.sql.` with full name, because:

- i want to call this out, eg this class is terrible to use, so needs attention
- class name in this package class with eg `java.util.`

i'm still fine with the import though.

Reply:
We could suppress the warning, but I think it's obvious here since we are calling `setTimestamp()`.

Reply:
What warning?